src/steps/readme.md

```diff
@@ -7,5 +7,6 @@ This GitHub action is following these steps in secuencial order:
 1. Create a compliance folder on the root of the repo
```

Comment:
I just noticed there's a typo on line 5 (before the bit you just wrote): 
It should say sequential rather than secuencial

Reply:
Thanks